### ISSUE TITLE
greetings fails to build

### ISSUE THREAD

**Issue by macmarrum** [freeplane/gradle-plugin#7]
`greetings` fails to build

Steps to reproduce:

```bash
export FREEPLANE_DIR=/home/data/git-repos/freeplane-macmarrum/BIN
git clone https://github.com/freeplane/gradle-plugin.git
cd gradle-plugin/examples/greetings
./gradlew build
```

The result

```
> Task :compileTestGroovy FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestGroovy'.
> Cannot infer Groovy class path because no Groovy Jar was found on class path: [/home/data/git-repos/gradle-plugin/examples/greetings/build/classes/java/main, /home/data/git-repos/gradle-plugin/examples/greetings/build/classes/groovy/main, /home/data/git-repos/gradle-plugin/examples/greetings/build/resources/main, /home/data/git-repos/gradle-plugin/examples/greetings/build/classes/java/test]
```

---

`greetings` builds OK after rolling back to d2f888cc
```bash
git checkout d2f888cc
./gradlew build
```

**Comment by dpolivaev** [comment]
fixed and version 0.7 released
I have only checked that the build works but not the produced add-on, let me know if it works or not.

**Comment by macmarrum** [comment]
Thanks
Tested OK :heavy_check_mark:
